MDNetworking/MDGameSession.cs

```diff
@@ -765,5 +765,34 @@ public void NotifyPlayerNameChanged(int peerId)
         {
             OnPlayerNameChanged(peerId);
         }
+
+        public bool ChangeNetworkMaster(Node Node, int NewNetworkMaster)
+        {
+            if (!GetAllPeerIds().Contains(NewNetworkMaster))
+            {
+                // Invalid network master
+                return false;
+            }
+            
+            // Only server can change network master, we could also allow server?
```

Comment:
confused by the second part of this comment. Is this meant to say we could also allow clients? I think keeping it server-only is a good idea

Reply:
Yeah that was supposed to say client, but I don't think we want client to do that so I will remove that part of the comment.